services/QuillLMS/app/assets/stylesheets/pages/unit_templates/unit_template_minis/unit_template_minis.scss

```diff
@@ -36,10 +40,20 @@
         width: 140px;
       }
     }
+    .dropdown-container:nth-of-type(2) {
+      width: 325px;
+    }
+  }
+
+  .pack-type-header {
+    font-weight: 700;
+    font-size: 20px;
   }
 
   .type-options {
     display: flex;
+    width: 668px;
```

Comment:
What happens to this on viewpoints smaller than 668px?

Reply:
it's not rendered on mobile: https://github.com/empirical-org/Empirical-Core/pull/9773/files#diff-43f845b2854ea0b7e27254f55b2f02f4256064b6ce439954dec9a044c7c11966R229

Reply:
it's not rendered below 770px: https://github.com/empirical-org/Empirical-Core/pull/9773/files#diff-43f845b2854ea0b7e27254f55b2f02f4256064b6ce439954dec9a044c7c11966R229